components/icons/InverterGraphicIcon.tsx

```diff
@@ -0,0 +1,37 @@
+const InverterGraphicIcon: React.FC = ({}) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="261"
+    height="99"
+    viewBox="0 0 261 99"
+    fill="none"
+  >
+    <circle cx="130" cy="49.5" r="49.5" fill="white" fill-opacity="0.06" />
+    <path
+      d="M3.08301 63.3333L5.66634 50H26.333L28.9163 63.3333H3.08301ZM4.37467 38.6666V36.6666H8.24967V38.6666H4.37467ZM5.47259 61.3333H15.0309V57.6666H6.18301L5.47259 61.3333ZM8.05592 46L6.66738 44.6L9.41217 41.7666L10.8007 43.1666L8.05592 46ZM6.57051 55.6666H15.0309V52H7.28092L6.57051 55.6666ZM15.9997 42.6666C14.4066 42.6666 13.0611 42.0833 11.9632 40.9166C10.8653 39.75 10.2733 38.3333 10.1872 36.6666H12.1247C12.1893 37.7777 12.5875 38.7222 13.3195 39.5C14.0514 40.2777 14.9448 40.6666 15.9997 40.6666C17.0545 40.6666 17.9479 40.2777 18.6799 39.5C19.4118 38.7222 19.8101 37.7777 19.8747 36.6666H21.8122C21.7261 38.3333 21.134 39.75 20.0361 40.9166C18.9382 42.0833 17.5927 42.6666 15.9997 42.6666ZM15.0309 48.6666V44.6666H16.9684V48.6666H15.0309ZM16.9684 61.3333H26.5268L25.8163 57.6666H16.9684V61.3333ZM16.9684 55.6666H25.4288L24.7184 52H16.9684V55.6666ZM23.9434 45.9666L21.1986 43.1333L22.5549 41.7333L25.332 44.5333L23.9434 45.9666ZM23.7497 38.6666V36.6666H27.6247V38.6666H23.7497Z"
+      fill="#FFD053"
+    />
+    <path
+      d="M54.5 50H85.5"
+      stroke="#FFD053"
+      stroke-width="3"
+      stroke-linecap="round"
```

Comment:
change these so react doesn't get mad at us 
```suggestion
      strokeWidth="3"
      strokeLinecap="round"
```